ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -6,6 +6,41 @@ include 'pan/types';
 include 'quattor/functions/network';
 
 @documentation{
+  IPv4/IPv6 prefix notation
+}
+function is_ip_prefix = {
```

Comment:
there is a `is_ipv4_prefix_length` and similar one for ipv6 in `pan/types` in template library core. this function should be moved there